src/create-verify-archive.js

```diff
@@ -76,6 +77,49 @@ exports.createCommand = function*(argv) {
     var absOutDir = path.resolve(outDir);
 
     yield repoutil.forEachRepo(repos, function*(repo) {
+        if(isWin) {
```

Comment:
Why the windows check?
Can a mac or linux users have a bad git config? and then the check will not run for them.